docs/unified-test-documentation/dasharo-compatibility/315-network-boot.md

```diff
@@ -1,13 +1,105 @@
 # Dasharo Compatibility: Network Boot
 
-## Test cases
+## PXE001.001 Dasharo Network Boot is available
 
-### PXE001.001 iPXE network boot
+**Test description**
+
+This test aims to verify that the `iPXE Network boot` option is available and
```

Comment:
In my opinion in the `Test description section` we shouldn't use a direct reference to the user.

So this point should be written as follows:
`This test aims to verify, that the iPXE Network boot is available in the Setup Menu and whether, after selecting this boot option Dasharo Network Boot Menu is displayed.`

Reply:
applied